src/lib/middleware/response-time-metrics.ts

```diff
@@ -66,6 +66,11 @@ export function responseTimeMetrics(
                 req.query.appName;
         }
 
+        const connectionId = req.headers['x-unleash-connection-id'];
```

Comment:
this is where it starts. on each detected connection id received we let the interested parties know